app/parsers/pbcore_xml_parser.rb

```diff
@@ -74,55 +69,56 @@ def create_works
     status_info(e)
   end
 
+  def recreate_entries
+    self.record_objects = []
+    records.each_with_index do |file, index|
+      set_objects(file, index).each do |record|
+        break if limit_reached?(limit, index)
+
+        seen[record[work_identifier]] = true
+        new_entry = find_or_create_entry(entry_class, record[work_identifier], 'Bulkrax::Importer', record.compact)
+      end
+      increment_counters(index)
+    end
+    importer.record_status
+  rescue StandardError => e
+    status_info(e)
+  end
+
+  def reload_works
+    self.record_objects = []
+    records.each_with_index do |file, index|
+      set_objects(file, index).each do |record|
+        break if limit_reached?(limit, index)
+        seen[record[work_identifier]] = true
+      end
+      increment_counters(index)
+    end
+  rescue StandardError => e
+    status_info(e)
+  end
+
```

Comment:
```suggestion
```

Reply:
I didn't see either of these methods being called in the changed code. were they left over from our mobbing session? or did I overlook something?

Reply:
these are super useful for debugging and fixing things. I need one of them for the coming bulkrax entry migration script. I've added comments trying to justify their existence. Does that seem like enough to keep them around?

Reply:
yes! that makes sense.